cunumeric/deferred.py

```diff
@@ -1125,13 +1131,15 @@ def convert(self, rhs: Any, warn: bool = True) -> None:
                 category=UserWarning,
             )
 
+        if nan_op is None:
```

Comment:
this if statement becomes unnecessary if you make the edit suggested above.

Reply:
bd361b6dc3a341df6a95f957f32321f70827ad38